api_code/db/2_db_populate.py

```diff
@@ -0,0 +1,138 @@
+'''
+    Author: Kyle Simpson
+    Description: Populate the provided database with the 
+                 VProgramDocuments txt file in ./gfatm_api/db_data/
+'''
+
+import json
+import sys
+import pandas as pd
+from db_utils import create_connection, create_table, insert_into_table
+
+prog_docs_location = r"C:\Users\Kyle\Desktop\api\gfatm_api\db_data\raw_program_docs.json"
+
+
+
+def load_prog_docs():
+    data = []
+    with open(prog_docs_location) as f:
+        data = json.load(f)['value']
+        data = pd.DataFrame.from_dict(data)
+
+    return(data)
+
+
+
+def create_base_table(conn):
+    '''
+        Description: Define the SQL to create the program_documents table, and call the
+                     create_table() function to execute the command
+    '''
+    prog_docs_sql = """ CREATE TABLE IF NOT EXISTS program_documents (

```

Comment:
In a more sophisticated application these should be the proper data types, but in this application there was loads of missing data so populating everything with strings was simpler.